src/libpgmoneta/utils.c

```diff
@@ -3003,6 +3004,128 @@ pgmoneta_is_file_archive(char* file_path)
    return false;
 }
 
+/* Parser for pgmoneta-cli amd pgmoneta-admin commands */
+bool
+parse_command(int argc,
+              char** argv,
+              int offset,
+              struct pgmoneta_parsed_command* parsed,
+              const struct pgmoneta_command command_table[],
+              size_t command_count)
+{
+#define EMPTY_STR(_s) (_s[0] == 0)
```

Comment:
I think you should move this to `pgmoneta.h`

Reply:
Just the macro, not the whole parse_command function, right?

Reply:
Just macro